tests/StreamSelectLoopTest.php

```diff
@@ -51,9 +51,9 @@ public function testStopShouldPreventRunFromBlocking($timeLimit = 0.005)
     public function signalProvider()
     {
         return [
-            ['SIGUSR1', SIGUSR1],
-            ['SIGHUP', SIGHUP],
-            ['SIGTERM', SIGTERM],
+            ['SIGUSR1', defined('SIGUSR1') ? SIGUSR1 : 10],
+            ['SIGHUP', defined('SIGHUP') ? SIGHUP : 1],
+            ['SIGTERM', defined('SIGTERM') ? SIGTERM : 15],
```

Comment:
This looks okay to me. My suggestion would have been to only return the constant names here and then use `constant()` to access their values within the test instead. This way we would not have to provide any defaults at all and could still skip the tests (possibly via `defined()`). What do you think about this?